FIO/remote-local-volumes/README.md

```diff
@@ -1,10 +1,12 @@
 # Create jobs
 
+## StorageOS FIO tests
+
 The volumes in this tests will be created without node selectors or hints,
```

Comment:
We cannot have a benchmark job that produces non-deterministic placement. Can we rename this entire section to 'remote-volumes', and do the same calculations on placement as for local-volumes but invert the logic so we put FIO on a guaranteed remote node?